modules/src/sensfusion6.c

```diff
@@ -29,6 +29,8 @@
 #include "imu.h"
 #include "param.h"
 
+#define M_PI_F ((float) M_PI)

```

Comment:
This might be a valuable macro to centralize, but I wasn't sure where you'd want it placed.  If you have suggestions....